### PR TITLE
Updated blog Astro 0.21 Preview article

### PR DESCRIPTION
**Pull request by mcfadyeni**
## Changes
- Improved install instructions and removed reference to merged GitHub next PR (#1406).

## Testing
- Blog text change only.

## Docs
- Docs not updated, only blog.

### CODE REVIEW

File: www/src/pages/blog/astro-021-preview.astro [hunk 1]
```diff
@@ -34,7 +34,7 @@ let lang = 'en';
         - [Brand new features, like Components-in-Markdown](#components-in-markdown)
         - [A new system for HTML live-updating via HMR](#hmr-meet-html)
 
-        You can try out our latest release today [in the browser](https://gitpod.io#snapshot/5e7cf2f1-8108-4fa5-99d3-ed8de70d8c23) or by running `npm install astro@next--compiler` in a new project directory on your machine.
```
Comment (jasikpark) [inline www/src/pages/blog/astro-021-preview.astro:37]: You might want to update the gitpod url as well - Maybe https://gitpod.io/#https://github.com/snowpackjs/astro/

Comment (mcfadyeni) [reply www/src/pages/blog/astro-021-preview.astro:37]: Will do, thanks!

**Comment by matthewp** [comment]
If you rebase with `main` this should pass again now.

**Comment by FredKSchott** [comment]
Thanks for the contribution! I think we'll leave this blog-post as is. Tomorrow `next` is becoming main and then this blog post would need to be updated again anyway.

**Pull request closed without merging**